apps/src/lib/config/ethereum_bridge/mod.rs

```diff
@@ -1,2 +1,37 @@
 pub mod ledger;
-pub mod params;
+
+#[cfg(test)]
+mod tests {
+    use eyre::Result;
+    use namada::ledger::eth_bridge::parameters::{
+        ContractVersion, Contracts, EthereumBridgeConfig, MinimumConfirmations,
+        UpgradeableContract,
+    };
+    use namada::types::ethereum_events::EthAddress;
+
+    /// Ensure we can serialize and deserialize a [`Config`] struct to and from
+    /// TOML. This can fail if complex fields are ordered before simple fields
+    /// in any of the config structs.
+    #[test]
+    fn test_round_trip_toml_serde() -> Result<()> {
```

Comment:
this test should be moved along with the `EthereumBridgeConfig`?

Reply:
Makes sense. Will do.